src/daft-plan/src/logical_ops/source.rs

```diff
@@ -14,6 +15,8 @@ pub struct Source {
     /// Information about the source data location.
     pub source_info: Arc<SourceInfo>,
 
+    // TODO(Clark): Replace these pushdown fields with the Pushdown struct, where the Pushdown struct would exist
+    // on the LegacyExternalInfo struct in SourceInfo.
```

Comment:
@jaychia Btw the TODO in the PR description "Consolidate filter/limit pushdowns to use the same Pushdown struct." is referring to this, which still should be done but definitely doesn't need to block the merging of the PR IMO.